python/paddle/fluid/tests/unittests/dygraph_to_static/test_mnist.py

```diff
@@ -196,34 +190,46 @@ def train(self, to_static=False):
                         start = time()
                     if batch_id == 50:
                         mnist.eval()
-                        prediction, acc, avg_loss = mnist(img, label)
+                        prediction = mnist(img)
                         loss_data.append(avg_loss.numpy()[0])
-                        self.check_save_inference_model([dy_x_data, y_data],
-                                                        prog_trans, to_static,
-                                                        prediction)
+                        self.check_jit_save_load(mnist, [dy_x_data], to_static,
+                                                 prediction)
                         break
         return loss_data
 
-    def check_save_inference_model(self, inputs, prog_trans, to_static, gt_out):
+    def check_jit_save_load(self, model, inputs, to_static, gt_out):
```

Comment:
Not for this PR, but a knowledge for you in the future:

When you deprecated `save_inference_model` and create a new API, you should add test for the new API, not replacing old test. In this case we can know the limitation of `save_inference_model` during unittest.

Reply:
done, thx! revert original unnittest change, and add new test